cmd/virt-launcher/entrypoint.sh

```diff
@@ -1,2 +1,29 @@
 #!/bin/bash
 ./virt-launcher $@
+rc=$?
+
+echo "virt-launcher exited with code $rc"
+
+# if the qemu pid outlives virt-launcher because virt-launcher
+# segfaulted/panicked/etc... then make sure we perform a sane
+# shutdown of the qemu process before exitting. 
+qemu_pid=$(pgrep -u qemu)
+if [ -n "$qemu_pid" ]; then
+	echo "qemu pid outlived virt-launcher process. Sending SIGTERM"
+	kill -SIGTERM $qemu_pid
+
+	# give the pid 10 seconds to exit. 
+	for x in $(seq 1 10); do
+		if ! [ -d /proc/$qemu_pid ]; then
+			echo "qemu pid exited after after SIGTERM"
```

Comment:
Would it be helpful to echo the actual pid itself here?

Reply:
added